packages/template-retail-react-app/app/pages/product-list/partials/page-designer-promotional-banner.jsx

```diff
@@ -41,7 +41,7 @@ const PageDesignerPromotionalBanner = () => {
                 </Box>
             )}
             {promoBannerMobile && !pageErrorMobile && (
-                <Box display={{base: 'block', md: 'none'}}>
+                <Box display={{base: 'block', sm: 'none'}}>
```

Comment:
changed to display the mobile promo banner when display is `sm` so i could upload a better sized image to improve lighthouse perf tests